website/yarn.lock

```diff
@@ -112,7 +113,16 @@
     source-map "^0.5.0"
     trim-right "^1.0.1"
 
-"@babel/generator@^7.19.0", "@babel/generator@^7.8.4":
+"@babel/generator@^7.12.5", "@babel/generator@^7.20.7":
+  version "7.20.7"
+  resolved "https://registry.yarnpkg.com/@babel/generator/-/generator-7.20.7.tgz#f8ef57c8242665c5929fe2e8d82ba75460187b4a"
```

Comment:
You may want to run [`yarn-deduplicate`](https://github.com/scinos/yarn-deduplicate).

Reply:
good idea, done